Description: Tweak build system
 We need to make sure the ChangeLog is not deleted, because
 otherwise building twice in a row would fail.
Forwarded: not-needed
Author: Andrea Bolognani <eof@kiyuko.org>
Last-Update: 2018-02-10
Index: beef.git/Makefile.am
===================================================================
--- beef.git.orig/Makefile.am
+++ beef.git/Makefile.am
@@ -18,7 +18,6 @@ EXTRA_DIST = \
 	$(NULL)
 
 CLEANFILES = \
-	ChangeLog \
 	$(NULL)
 
 ChangeLog:
